docker-compose.yml

```diff
@@ -47,15 +47,90 @@ services:
       n1l1tos1:
         ipv4_address: 10.168.1.5
       n2htoleaf1:
-      n3htoleafbn1:
-      n5h1tol1r:
       n6h1tol1y:
     command: |
       sh -x -c 'touch /etc/frr/vtysh.conf && \
             sed -i "s/bgpd=no/bgpd=yes/g" /etc/frr/daemons && \
             sed -i "s/127.0.0.1/0.0.0.0/g" /etc/frr/daemons && \
             ip link add name lo0 type dummy && \
             ifconfig lo0 10.0.0.2 netmask 255.255.255.255 up && \
+
```

Comment:
I do not see anywhere configured loopbackIPs in the VRFs. I think that is needed as in our API for now the VRFs loopback APIs are required so it would be a good testing point of our API

Reply:
Are you expecting to do some OAM operation (Ping to the loopback IP in the VRF) here?

Reply:
good idea, let's add ping

Reply:
Added a loopback per vrf in leaf1.